### PR TITLE
2023-03-29-newsletter-translate-in-french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by Copinmalin** [comment]
Thanks @bitschmidty for your confirmation

File: _posts/fr/newsletters/2023-03-29-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    entièrement à la partie qui est en violation du protocole, il n'y
+    a aucun risque de sécurité à externaliser les informations sur les
+    dépenses à partir de cette transaction).
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27278,26531,5898,5986,2616,2024,1737,4608,1425,22,18861" %}
+[lnd v0.16.0-beta.rc5]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta.rc5
+[bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
+[rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
+[news230 tp]: /en/newsletters/2022/12/14/#factory-optimized-ln-protocol-proposal
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-03-29-newsletter.md:389]:
```suggestion
[news230 tp]: /fr/newsletters/2022/12/14/#proposition-d-optimisation-du-protocole-d-usines-a-canaux-ln
```

File: _posts/fr/newsletters/2023-03-29-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+[obeirne selfish]: https://twitter.com/jamesob/status/1637198454899220485
+[sanders requests]: https://twitter.com/theinstagibbs/status/1637235436849442817
+[news133 usdt]: /en/newsletters/2021/01/27/#bitcoin-core-19866
+[libwally]: https://github.com/ElementsProject/libwally-core
+[news128 psbt2]: /en/newsletters/2020/12/16/#new-psbt-version-proposed
+[tunable penalties]: https://github.com/JohnLaw2/ln-tunable-penalties
+[news238 libwally]: /en/newsletters/2023/02/15/#libwally-0-8-8-released
+[rust-bitcoin.org]: https://rust-bitcoin.org/
+[rb rn]: https://github.com/harding/rust-bitcoin/blob/bbda9599fa32936f31472620d014893fda17d8c3/bitcoin/CHANGELOG.md#030---2023-03-21-the-first-crate-smashing-release
+[news243 bdk]: /en/newsletters/2023/03/22/#bdk-793
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-03-29-newsletter.md:406]:
```suggestion
[news243 bdk]: /fr/newsletters/2023/03/22/#bdk-793
```

File: _posts/fr/newsletters/2023-03-29-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+[news239 codex32]: /en/newsletters/2023/02/22/#proposed-bip-for-codex32-seed-encoding-scheme
+[law stranded post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-March/003886.html
+[law stranded paper]: https://github.com/JohnLaw2/ln-hierarchical-channels
+[obeirne selfish]: https://twitter.com/jamesob/status/1637198454899220485
+[sanders requests]: https://twitter.com/theinstagibbs/status/1637235436849442817
+[news133 usdt]: /en/newsletters/2021/01/27/#bitcoin-core-19866
+[libwally]: https://github.com/ElementsProject/libwally-core
+[news128 psbt2]: /en/newsletters/2020/12/16/#new-psbt-version-proposed
+[tunable penalties]: https://github.com/JohnLaw2/ln-tunable-penalties
+[news238 libwally]: /en/newsletters/2023/02/15/#libwally-0-8-8-released
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-03-29-newsletter.md:403]:
```suggestion
[news238 libwally]: /fr/newsletters/2023/02/15/#sortie-de-libwally-0-8-8
```

File: _posts/fr/newsletters/2023-03-29-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+{% include linkers/issues.md v=2 issues="27278,26531,5898,5986,2616,2024,1737,4608,1425,22,18861" %}
+[lnd v0.16.0-beta.rc5]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta.rc5
+[bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
+[rust bitcoin 0.30.0]: https://github.com/rust-bitcoin/rust-bitcoin/releases/tag/bitcoin-0.30.0
+[news230 tp]: /en/newsletters/2022/12/14/#factory-optimized-ln-protocol-proposal
+[channel factories paper]: https://tik-old.ee.ethz.ch/file//a20a865ce40d40c8f942cf206a7cba96/Scalable_Funding_Of_Blockchain_Micropayment_Networks%20(1).pdf
+[law factories]: https://raw.githubusercontent.com/JohnLaw2/ln-efficient-factories/main/efficientfactories10.pdf
+[news206 msat]: /en/newsletters/2022/06/29/#core-lightning-5306
+[rb sec]: https://github.com/rust-bitcoin/rust-bitcoin/blob/master/SECURITY.md
+[news239 codex32]: /en/newsletters/2023/02/22/#proposed-bip-for-codex32-seed-encoding-scheme
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-03-29-newsletter.md:394]:
```suggestion
[news239 codex32]: /fr/newsletters/2023/02/22/#proposition-de-bip-pour-le-systeme-d-encodage-des-semences-codex32
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Pull request merged**